docs/faq.md

```diff
@@ -109,7 +109,7 @@ instances and run the following commands:
 
 ## Failed to install
 
-Follow the same steps as specified in failed to update.
+Follow the same steps as specified in [failed to update](https://github.com/TypeStrong/atom-typescript/blob/master/docs/faq.md#failed-to-update).
```

Comment:
Since it's an internal link, it'd be arguably better to omit the page URL.
```suggestion
Follow the same steps as specified in [failed to update](#failed-to-update).
```

Reply:
Actually, will fix this on our end